pavelib/utils/test/utils.py

```diff
@@ -17,7 +17,7 @@ def clean_test_files():
     """
     Clean fixture files used by tests and .pyc files
     """
-    sh("git clean -fqdx test_root/logs test_root/data test_root/staticfiles test_root/uploads")
+    sh("git clean -fqdx test_root/logs test_root/data test_root/uploads")
```

Comment:
I see what you mean.... I'm really not sure, either.  I wonder if it is being handled by `runserver` even when using fasttest because bokchoy is run in debug mode ([see these lines](https://github.com/edx/edx-platform/blob/master/cms/envs/bok_choy.py#L83-L84))?